tracing/src/test/java/io/airlift/tracing/TestTracingModule.java

```diff
@@ -0,0 +1,82 @@
+package io.airlift.tracing;
+
+import io.airlift.bootstrap.Bootstrap;
+import io.airlift.node.testing.TestingNodeModule;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class TestTracingModule
+{
+    @Test
+    public void testCreateNoopTracing()
+    {
+        Bootstrap bootstrap = new Bootstrap(new TracingModule("airlift", "0.1"));
+        OpenTelemetry instance = bootstrap
+                .doNotInitializeLogging()
+                .initialize()
+                .getInstance(OpenTelemetry.class);
+
+        assertThat(instance).isEqualTo(OpenTelemetry.noop());
+    }
+
+    @Test
+    public void testCreateTracingWithDefaultConfig()
+    {
+        Bootstrap bootstrap = new Bootstrap(
+                new TracingModule("airlift", "0.1"),
+                new TestingNodeModule("test"));
+        OpenTelemetry instance = bootstrap
+                .doNotInitializeLogging()
+                .setRequiredConfigurationProperty("tracing.enabled", "true")
+                .initialize()
+                .getInstance(OpenTelemetry.class);
+
+        assertThat(instance).isNotEqualTo(OpenTelemetry.noop());
+        assertThat(instance).isInstanceOf(OpenTelemetrySdk.class);
+        assertThat(instance.toString())
+                .contains("BatchSpanProcessor{spanExporter=io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter");
+    }
+
+    @Test
+    public void testCreateTracingWithCustomSpanProcessor()
+    {
+        Bootstrap bootstrap = new Bootstrap(
+                new TracingModule(binder -> {
+                    binder.bind(SpanProcessor.class).toInstance(SimpleSpanProcessor.create(OtlpHttpSpanExporter.getDefault()));
```

Comment:
you can use `InMemorySpanExporter.create()`
and then verify the span processor is being used to export some spans

i think it would be cleaner than the instanceof and toString-based assertions below.